src/edu/washington/escience/myria/api/encoding/ParallelDatasetEncoding.java

```diff
@@ -0,0 +1,24 @@
+/**
+ *
+ */
+package edu.washington.escience.myria.api.encoding;
+
+import java.util.Set;
+
+import edu.washington.escience.myria.RelationKey;
+import edu.washington.escience.myria.Schema;
+import edu.washington.escience.myria.io.AmazonS3Source;
+
+/**
+ *
+ */
+public class ParallelDatasetEncoding extends MyriaApiEncoding {
+  @Required public RelationKey relationKey;
+  @Required public Schema schema;
+  @Required public AmazonS3Source s3Source;
+  public Character delimiter;
+  public Character escape;
+  public Integer numberOfSkippedLines;
+  public Character quote;
+  public Set<Integer> workers;
```

Comment:
`DatasetEncoding` (which this more or less copies) has a `partitionFunction` field that defaults to `RoundRobinPartitionFunction`. Should our range partitioning be considered equivalent to `RoundRobinPartitionFunction` for the purposes of the optimizer (i.e., it can't make any assumptions about partitioning based on tuple values), or should we introduce a new `RangePartitionFunction` (that would presumably be treated the same way by the optimizer), to make the distinction between range partitioning and round-robin partitioning clear? At least I think we need *some* partitioning information in the catalog.